packages/venia-ui/lib/drivers/adapter.js

```diff
@@ -38,6 +39,65 @@ persistCache({
     storage: window.localStorage
 });
 
+const initialData = {
+    __typename: 'CartData',
```

Comment:
I might be wrong on this kind of a use case. Let me know if that isn't how we should be doing it. Will make the changes.

Reply:
Yes, I am aware of that. But I was having a hard time writing the local resolvers and local state with the current cart. Hence I moved it to the top level. I was in a view that this code is only useful until the Gql team comes up with the support for gift options. Once that happens, this whole block of code is scrapped off.

Reply:
> this code is only useful until the Gql team comes up with the support for gift options.

True -- but that may be a very long time, and there are other advantages to containing the options or other cart-specific state within the cart. We wouldn't want other carts to start showing state from a previous one :o

Reply:
Done. Now the gift options are scoped to a cart id.